spec/rhc/commands/snapshot_spec.rb

```diff
@@ -32,14 +32,26 @@
 
     context 'when saving a snapshot' do
       before do
+        RHC::Helpers.should_receive(:agree).with(/Do you want to overwrite this file?/).and_return(true)
```

Comment:
Why is this needed? Are we assuming that `#{@app.name}.tar.gz` exists from a previous test?